libobs/obs-service.h

```diff
@@ -43,6 +43,7 @@ enum obs_service_connect_info {
 	OBS_SERVICE_CONNECT_INFO_PASSWORD = 6,
 	OBS_SERVICE_CONNECT_INFO_ENCRYPT_PASSPHRASE = 8,
 	OBS_SERVICE_CONNECT_INFO_BEARER_TOKEN = 10,
+	OBS_SERVICE_CONNECT_INFO_CONFIG_URL = 12,
```

Comment:
This does not look like being part of the eRTMP spec but something alike to a custom behavior for a specific service.

`obs_service_connect_info` is meant to try to abstract filed between protocols not service-specific/custom. behavior.

Reply:
Backing this up. This enum is intended to be used to describe parameters for configuring the protocol being used for streaming. The configuration URL used for Enhanced Broadcast is an out of scope of the RTMP protocol itself, just like  the URL `https://ingest.twitch.tv/ingests` is, so this system should not be used to pass this data around.

Reply:
moved this functionality in https://github.com/obsproject/obs-studio/pull/10543/commits/06d758a0d5bcc25f36376047da2c6d364ee519ef (<https://github.com/obsproject/obs-studio/pull/10543#event-12615445003>)

Reply:
This change was reverted, so this feedback was addressed. Resolving.